README.md

```diff
@@ -14,6 +14,9 @@ UCSD DAMS Vagrant Virtual Machine
 
 You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localhost`
 
+You can load test record to vagrant by running cURL commands:
```

Comment:
this is nice and concise :+1: 

Can you modify the wording slightly to be a bit more explicit? Something like: `You can load test records into DAMS Repository, installed in the Vagrant VM, using cURL:` 

It might also be nice to add a link to some examples, perhaps https://github.com/ucsdlib/damspas/tree/develop/spec/fixtures ?